azure-pipelines/compliance.yml

```diff
@@ -62,6 +62,16 @@ jobs:
     - task: CredScan@3
       inputs:
         scanFolder: '$(Build.SourcesDirectory)'
+    - task: APIScan@2
+      displayName: 'Run APIScan'
+      inputs:
+        softwareFolder: '$(Build.ArtifactStagingDirectory)/drop'
+        softwareName: 'Visual Studio Tools for Unreal Engine'
+        softwareVersionNum: '2.4'

```

Comment:
Do you think there's a way for us to keep this in sync with the value in the `.uplugin` file, which controls the submission to the Unreal Marketplace? Or do we even care? I'm not quite sure how this is used for the task itself.

Reply:
I'm not sure either. And since I don't know how to keep this in sync, we can just leave it like this for now.

Reply:
Actually seems like that value is not used.